docs/api.md

```diff
@@ -216,6 +216,41 @@ const normalizedData = normalize(data, tweet);
 }
 ```
 
+When passing the `idAttribute` a function, it should return the IDs value. 
+
+For Example:
+
+```js
+const myData = {
+  patrons: [
+    { id: 1, guest_id: null, name: "Esther" },
+    { id: 1, guest_id: 22, name: "Tom" },
+  ]
+};
+
+const patronsSchema = new schema.Entity('patrons', undefined, {
+  // idAttribute *functions* must return the ids **value** (not key)
+  idAttribute: value => value.guest_id ? `${value.id}-${value.guest_id}` : value.id,
+});
+
+normalize(myData, [patronsSchema]);
+```
+
+#### Output
+
+```js
+{
+    entities: {
+      patrons: {
+        1: { ... },
+        1-22: { ... },
```

Comment:
nit: make sure these are quoted as strings, otherwise that's invalid JS, ex: `'1-22': { ... }`